Automattic/jetpack
# WordAds: Set top AMP ad unit to fixed size of 300x250 pixels

#### Changes proposed in this Pull Request:

On mobile the top AMP ad slot was set to a size of 320x50. When filled with a larger ad the ad was being cropped, as AMP prevents visible ad slots from expanding in size.

The PR sets the the top AMP ad slot to a fixed size of 300x250 to prevent ads being cropped.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:






* Visit ads enabled site in AMP mode on mobile device (or simulated mobile browser)
* Verify that ad with class `top_amp` is set to a final size of 300x265 (the extra 15 pixels is for padding)
